traffic_ops/client/server_update_status.go

```diff
@@ -18,33 +18,32 @@ package client
 import (
 	"encoding/json"
 	"fmt"
-	"net"
 	"net/http"
 
 	"github.com/apache/trafficcontrol/lib/go-tc"
 )
 
-const (
-	APIUpdateServerStatus = apiBase + "/servers/%d/status"
-)
-
 // UpdateServerStatus updates a server's status and returns the response.
 func (to *Session) UpdateServerStatus(serverID int, req tc.ServerPutStatus) (*tc.Alerts, ReqInf, error) {
-	var remoteAddr net.Addr
+	reqInf := ReqInf{CacheHitStatus: CacheHitStatusMiss}
+
 	reqBody, err := json.Marshal(req)
-	reqInf := ReqInf{CacheHitStatus: CacheHitStatusMiss, RemoteAddr: remoteAddr}
 	if err != nil {
 		return nil, reqInf, err
 	}
-	path := fmt.Sprintf(APIUpdateServerStatus, serverID)
+
+	path := fmt.Sprintf(apiBase+"/servers/%d/status", serverID)
```

Comment:
bit of a nitpick but if you're already doing a `fmt.Sprintf` you could just fully utilize it like
```go
path := fmt.Sprintf("%s/servers/%d/status", apiBase, serverID)
```
 instead of concatenating a string first. I'm sure the compiler will sort it out, but it's just weird to see two different string building methods in the same expression.